src/index.js

```diff
@@ -35,6 +35,14 @@ client.getCommand = function(name) {
 
 require("./commands")(client);
 
+function suggestCommands(cmd, cmdList) {
+  var suggestion = didYouMean(cmd, cmdList);
```

Comment:
The `suggestCommands()` method logs those warnings as per now and returning the suggestion doesn't make sense right :thinking: